onebusaway-android/src/androidTest/java/org/onebusaway/android/nav/test/NavigationTest.java

```diff
@@ -101,23 +101,21 @@ public void testTrip2C() throws IOException {
      * Destination Stop: Dale Mabry Hwy @ Hudson Ln @ Taco Bell
      * Recorded In: Bus (Route 36) 4 stops
      * Device Used: Nexus 5
-     * FIXME - Currently fails with "Pull the cord triggered too soon"
      */
     @Test
     public void testTrip3() throws IOException {
-        runSimulation("nav_trip3", 95, 115);
+        runSimulation("nav_trip3", 95, 109);
```

Comment:
It looks like the algorithm did indeed trigger the notice to the user to "pull the cord" too soon.  Looking at the below screenshot, with the bus traveling towards the top of the screenshot, it looks like ID 111 is the point that should have triggered the "pull the cord now" alert, if our algorithm worked perfectly.  It looks like 109 is the green alert balloon where the algorithm actually triggered the "pull the cord now" alert, but this was before the previous stop, which is too early.

![image](https://user-images.githubusercontent.com/928045/45648822-9fe1bf00-ba97-11e8-8bfa-dd2fdd731f05.png)